Exercises/recursion/recursion.js

```diff
@@ -0,0 +1,10 @@
+//1. Make a function which will flat passed array i. e. : ``[1, [2, 3], [4, [5]]]` ->  [1, 2, 3, 4, 5].
+
+const newArr = [];
```

Comment:
It's a bad practice to modify and return outside scope variable when you can avoid it.
Try to rewrite this with making this function pure.

Reply:
BTW try to call your function twice